scripts/drop-database.js

```diff
@@ -0,0 +1,41 @@
+const { Client } = require('pg')
```

Comment:
As with `create-database`, you should make sure to run prettier across this to bring it up to standard